src/utils/hooks/useBrowserTabUpdate.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import app_config from '~core/app_config';
+import { trimLinkIfInDev } from '~utils/common';
+
+export function useBrowserTabUpdateEffect(iconPath?: string) {
```

Comment:
tried to find a way to just inline these icons on build step
it seems that it's still in progress in vite https://github.com/vitejs/vite/pull/8717